clients/dlp/lib/google_api/dlp/v2/model/google_privacy_dlp_v2_action.ex

```diff
@@ -12,20 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# NOTE: This class is auto generated by the swagger code generator program.
-# https://github.com/swagger-api/swagger-codegen.git
+# NOTE: This class is auto generated by the elixir code generator program.
 # Do not edit the class manually.
 
 defmodule GoogleApi.DLP.V2.Model.GooglePrivacyDlpV2Action do
   @moduledoc """
-  A task to execute on the completion of a job. See https://cloud.google.com/dlp/docs/concepts-actions to learn more.
+  A task to execute on the completion of a job.
+  See https://cloud.google.com/dlp/docs/concepts-actions to learn more.
 
   ## Attributes
 
-  - jobNotificationEmails (GooglePrivacyDlpV2JobNotificationEmails): Enable email notification to project owners and editors on job&#39;s completion/failure. Defaults to: `null`.
-  - pubSub (GooglePrivacyDlpV2PublishToPubSub): Publish a notification to a pubsub topic. Defaults to: `null`.
-  - publishSummaryToCscc (GooglePrivacyDlpV2PublishSummaryToCscc): Publish summary to Cloud Security Command Center (Alpha). Defaults to: `null`.
-  - saveFindings (GooglePrivacyDlpV2SaveFindings): Save resulting findings in a provided location. Defaults to: `null`.
+  - jobNotificationEmails (GoogleApi.DLP.V2.Model.GooglePrivacyDlpV2JobNotificationEmails.t): Enable email notification to project owners and editors on job's
+  completion/failure. Defaults to `nil`.
```

Comment:
@chingor Curious why this now includes a newline. Did the upstream description text change?

Reply:
Huh, okay, looking at other examples, it looks like the new generator is now preserving newlines in the documentation strings rather than squashing them together. Probably in most cases this is what we want.